### PR TITLE
Adds support for subtitles

### PR DESCRIPTION
**Pull request by wa4557**
Support for playing subtitles alongside your video files. Note that this only works if you have CORS enabled both on your server where the subtitles and the video files are stored. 

I have not figured out yet how to change the subtitles while the video is still playing

### CODE REVIEW

File: pychromecast/controllers/media.py [hunk 1]
```diff
@@ -443,7 +448,17 @@ def play_media(self, url, content_type, title=None, thumb=None,
...
-
+        if subtitles:
+            sub_msg = {
+                'trackId': subtitle_id, 
+                'trackContentId': subtitles,
+                'language': subtitles_lang, 
+                'subtype': 'SUBTITLES',
+                'type': 'TEXT',
+                'ttrackContentType': subtitles_mime, 
+                'name': subtitles_lang + '-' + str(subtitle_id) + ' Subtitle'
```
Comment (balloob) [inline pychromecast/controllers/media.py]: Please use string formatting.

**Comment by balloob** [comment]
Looks good. One minor comment and you have a few trailing whitespaces (see build errors). Can merge when both addressed.

**Comment by wa4557** [comment]
Alright, pushed :+1:

File: pychromecast/controllers/media.py [hunk 2]
```diff
@@ -443,7 +448,17 @@ def play_media(self, url, content_type, title=None, thumb=None,
...
-
+        if subtitles:
+            sub_msg = {
+                'trackId': subtitle_id, 
+                'trackContentId': subtitles,
+                'language': subtitles_lang, 
+                'subtype': 'SUBTITLES',
+                'type': 'TEXT',
+                'ttrackContentType': subtitles_mime, 
+                'name': "%s-%d Subtitle"%(subtitles_lang, subtitle_id)
```
Comment (balloob) [inline pychromecast/controllers/media.py]: Sorry, I wasn't clear. Please use the modern string formatting. 

```python
"{} - {} Subtitle".format(subtitles_lang, subtitle_id)
```

**Comment by balloob** [comment]
[Still linting build errors](https://travis-ci.org/balloob/pychromecast/jobs/153959053#L347)…

**Comment by wa4557** [comment]
Sorry, was not looking closely enugh

**Comment by balloob** [comment]
Looks good 🐬

**Pull request merged**